launcher/actions.py

```diff
@@ -11,8 +11,7 @@ class ProjectManagerAction(avalon.api.Action):
     def is_compatible(self, session):
         return "AVALON_PROJECT" in session
 
-    def process(self, session, **kwargs):
-
+    def process(self, session):
```

Comment:
This change is invalid, the method's signature should include `**kwargs`. See [here](https://github.com/Colorbleed/core/blob/master/avalon/pipeline.py#L247)